source/_components/binary_sensor.octoprint.markdown

```diff
@@ -25,14 +25,26 @@ To set it up, add the following information to your `configuration.yaml` file:
 ```yaml
 binary_sensor:
   - platform: octoprint
+    name: YOUR_OCTOPRINT_NAME
     monitored_conditions:
       - Printing
       - Printing Error
 ```
 
-Configuration variables:
+{% configuration %}
+name:
+  description: Name to use in the frontend.
+  required: true
+  type: string
+  default: Octoprint
+monitored_conditions:
+  description: Conditions to monitor.
+  required: true
+  type: list
+{% endconfiguration %}
 
+Configuration variables:
```

Comment:
This is now duplicate? Since you've already created a new list just above this, using the new formatting.